apps/next/tailwind.config.js

```diff
@@ -102,13 +102,9 @@ module.exports = {
         ".text-3xl": textSizes["text-3xl"],
         ".text-4xl": textSizes["text-4xl"],
         ".safe-top": {
-          paddingTop: "constant(safe-area-inset-top)",
-          // eslint-disable-next-line no-dupe-keys
           paddingTop: "env(safe-area-inset-top)",
         },
         ".safe-bottom": {
-          paddingBottom: "constant(safe-area-inset-bottom)",
```

Comment:
remove this because `constant` only support iOS <= `11.2`, but now nobody using iOS 11 basically

Reply:
Yes, that is fine.